modules/ting/ting.module

```diff
@@ -527,7 +527,7 @@ function ting_ding_entity_is($object, $class) {
   if ($class == 'reservable' || $class == 'cartable') {
     $sources = variable_get('ting_reservable_sources', _ting_default_reservable_sources());
     $types = variable_get('ting_reservable_types', _ting_default_reservable_types());
-    if (in_array(drupal_strtolower($object->ac_source), $sources) && in_array(drupal_strtolower($object->type), $types)) {
```

Comment:
Any reason to change this from magic property to method? It does not seem as if we do so everywhere else.

Reply:
True - might as well keep it consistent